src/test/java/com/epam/reportportal/restendpoint/http/mock/RestEndpointProxyTest.java

```diff
@@ -91,7 +91,7 @@ public void testPost() throws IOException, InterruptedException {
 	@Test
 	public void testPostVoid() throws IOException, InterruptedException {
 		server.enqueue(prepareResponse(""));
-		Maybe<Void> to = restInterface.postVoid(String.format(SERIALIZED_STRING_PATTERN, 100, "test string"));
+		Maybe<Void> to = restInterface.postVoid(String.format(SERIALIZED_STRING_PATTERN, 100, "test string")).cache();
```

Comment:
I would say that this line:
https://github.com/reportportal/restendpoint/pull/5/files#diff-bd3f581090a1f726ba375e103fac6cdbR103
```to.blockingGet();```
Was forgot here during debugging. Rather than using request cache was intentionally for the test.

Reply:
@avarabyeu What do you think?